### ISSUE TITLE
[Git] add .gitattributes

### ISSUE THREAD

**Issue by OTKRyu** [Samsung/ONE-vscode#175]
.gitattributes file is for everyone who wants to contribute this prooject to have same git settings.
I notice this need of .gitattributes because of CRLF and LF.
Depends on which os users use, CRLF and LF cause changes but git couldn't catch that CRLF and LF are not meaningful change for human.
So in github, there are so many useless change are tracked. I want to prevent this situation and .gitattributes file can do what i want.
Before i actually make pr, i want to ask everyone this is useful or not

**Comment by seanshpark** [comment]
What exactly do you want to ask? It's a bit hard to catch your question.

**Comment by OTKRyu** [comment]
I want to ask adding .gitattributes file to this project is useful or not

**Comment by seanshpark** [comment]
Thanks for updating the message.

As this repo main development is done ine Linux, I want to use as `LF`, not `CRLF`
I'm not used to `.gitattributes` . 
Can you please provide how `.gitattributes` will solve your problem? And I'm not sure what exactly is your problem.

**Comment by OTKRyu** [comment]
<http://daplus.net/git-%EC%A4%84-%EB%81%9D-%EC%84%A4%EC%A0%95%EC%9D%84-%EB%B3%80%EA%B2%BD%ED%95%98%EB%8A%94-%EB%B0%A9%EB%B2%95/>
I saw this document about methods that prevent changing LF to CRLF by window.
If we set .gitattributes, .gitattributes prevent window users's commits changing LF to CRLF by window.
I know this project is going on linux, but if window user wants to contribute this project, I think this problem migth cause unconvenience.

**Comment by seanshpark** [comment]
Is your problem `My editor changes LF to CRLF` ?

**Comment by OTKRyu** [comment]
Yes. so i want something prevent this problem and i think .gitattributes can be answer.

**Comment by seanshpark** [comment]
> Yes. so i want something prevent this problem and i think .gitattributes can be answer.

Can you post a draft PR and some captures that will justify it?

**Comment by OTKRyu** [comment]
yes. i will make draft pr soon.
